tools/ts_uinput.c

```diff
@@ -872,7 +896,7 @@ int main(int argc, char **argv)
 	free(testsample_p);
 
 	/* blocking setup for production run */
-	data.ts = ts_setup(data.input_name, 0);
+	data.ts = ts_setup(data.input_name, 1);
```

Comment:
why do you change this and open the touchscreen non-blocking now? thanks

Reply:
I tested it on my Raspberry Pi, If I don't change this to non-blocking mode, the update would not effect until your next touch.
I also wrote anther version use blocking mode, multi-thread way to do this, Whenever I detect that calibfile has been modified, I kill the process-thread, reconfig, and restart the process-thread again. but I am afraid that kill the thread may cause some unknown problems.